src/rtprecv.c

```diff
@@ -322,6 +323,26 @@ static void decode_tmr(void *arg)
 }
 
 
+static void rtprecv_resync(struct rtp_receiver *rx,
+			 const struct rtp_header *hdr)
+{
+	if (!rx || !hdr)
+		return;
+
+	jbuf_flush(rx->jbuf);
+
+	mtx_lock(rx->mtx);
+	rx->ssrc = hdr->ssrc;
+	rx->ssrc_set = true;
+	rx->pseq = (uint16_t)(hdr->seq - 1);
+	rx->pseq_set = true;
+	rx->pt = -1;
+	rx->pt_tel = 0;
+	rx->ssrc_changed = true;
+	mtx_unlock(rx->mtx);
+}
+
+
```

Comment:
Now there is some redundant code in https://github.com/Turkish/baresip/blob/099d60b06ffa562005c9c595b689c2495d374d88/src/rtprecv.c#L463-L465